snippets/serializeForm.md

```diff
@@ -0,0 +1,21 @@
+### serializeForm
+
+Serializes a form into an object.
```

Comment:
The two snippets have the same description, please change one of them to clarify the difference between the two.

Reply:
ok, i fix it.